hasjob/views/admin_filterset.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+from flask import flash, g
+from coaster.views import route, viewdata, UrlForView, ModelView
+from baseframe import __
+from baseframe.forms import render_form, render_redirect
+from .. import app, lastuser
+from ..models import db, Filterset
+from ..forms import FiltersetForm
+from .admin import AdminView
+
+
+@route('/f')
+class AdminFilterset(AdminView):
+    __decorators__ = [lastuser.requires_permission('siteadmin')]
```

Comment:
We should switch to `requires_permission('boardadmin')` and grant that permission from `Board.permissions`

Reply:
Actually, `boardadmin` is more appropriately a role than a permission. The permission should be `edit-filterset`.